opm/simulators/linalg/bda/BdaBridge.hpp

```diff
@@ -45,10 +45,11 @@ template <class BridgeMatrix, class BridgeVector, int block_size>
 class BdaBridge
 {
 private:
-    std::unique_ptr<bda::BdaSolver<block_size> > backend;
     bool use_gpu = false;
 
 public:
+    std::unique_ptr<bda::BdaSolver<block_size> > backend;
```

Comment:
It would be really nice, if we could prevent this from being public. If the access is needed maybe can add a getter:
```
const bda::BdaSolver<block_size>& getBackend() const;
```

Reply:
That makes sense. I'll try to fix it